code/modules/mob/living/carbon/human/species_types/plasmamen.dm

```diff
@@ -22,6 +22,15 @@
 	changesource_flags = MIRROR_BADMIN | WABBAJACK | MIRROR_PRIDE | MIRROR_MAGIC
 	outfit_important_for_life = /datum/outfit/plasmaman
 
+	// Body tempurature for Plasmen is much lower
+	// they can handle colder environments
+	// but they are hurt at hot temps faster as it is harder to hold their form
+	// and they will regain tempurature more slowly in some situations
+	bodytemp_normal = (BODYTEMP_NORMAL - 40)
+	bodytemp_autorecovery_min = 2
+	bodytemp_heat_damage_limit = (BODYTEMP_HEAT_DAMAGE_LIMIT - 20) // about 40C
+	bodytemp_cold_damage_limit = (BODYTEMP_COLD_DAMAGE_LIMIT - 50) // about -50c

```

Comment:
... you do know that plasmamen are immune to damage and slowdown from the cold (they have the TRAIT_RESISTCOLD trait), right?

Reply:
That is why they like having a cold body temperature.

Reply:
/datum/species/proc/handle_body_temperature also handles the resist traits.
if you mean why does the number exist? well i picked a fair number and thought the traits would handle the rest.